README.md

```diff
@@ -33,7 +33,7 @@ Quick links:
 
 ### <a id="docs"></a>Documentation and getting help ###
 
-* There are Scaladoc API documentations for [the core library][core-api], which defines and implements the core types for streams and pulls, as well as the type aliases for pipes and sinks. [The `io` library][io-api] provides FS2 bindings for NIO-based file I/O and TCP/UDP networking.
+* There are Scaladoc API documentations for [the core library][api], which defines and implements the core types for streams and pulls, as well as the type aliases for pipes and sinks. [The `io` library][io-api] provides FS2 bindings for NIO-based file I/O and TCP/UDP networking.
```

Comment:
Actually the API docs are no longer split into core/io, it's all under the `[api]` link now. I suspect the second link is also broken.

Reply:
I have removed reference to that split. I made the phrasing of this bullet point similar to the one after it. Please let me know if you have other preferences/suggestions on wording.

Reply:
As there remained no references to `[io-api]` or to `[rx-api]`, I removed the Sonatype links to both.